src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -1703,6 +1703,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       <_ThisProjectBuildMetadata Include="$(MSBuildProjectFullPath)">
         <TargetFrameworks Condition="'$(TargetFrameworks)' != ''">$(TargetFrameworks)</TargetFrameworks>
         <TargetFrameworks Condition="'$(TargetFrameworks)' == ''">$(TargetFramework)</TargetFrameworks>
+        <TargetFrameworks Condition="'$(TargetFrameworks)' == '' and '$(TargetFramework)' == ''">$(TargetFrameworkMoniker)</TargetFrameworks>
```

Comment:
This probably needs to respect NuGetTargetMoniker for UWP at least where it is distinct from TargetFrameworkMoniker.  But see weirdness with old school PCL in ReferrringTargetFrameworkMonikerForProjectReferences (not sure if I got the exact mouthful right from memory).

Reply:
Nice catch.

https://github.com/Microsoft/msbuild/blob/d2e8794ee2d6c2002a248d70ee315fc8380d82e5/src/Tasks/Microsoft.Common.CurrentVersion.targets#L1583-L1593